app/renderer/src/reducers/copiedToClipboard.js

```diff
@@ -0,0 +1,21 @@
+import {
+  UPDATE_QUERY,
+  SELECT_ITEM,
+  SELECT_NEXT_ITEM,
+  SELECT_PREVIOUS_ITEM,
+  COPY_ITEM,
+} from '../actions/types';
+
+export default function (state = false, { type }) {
+  switch (type) {
+    case UPDATE_QUERY:
```

Comment:
@vutran Can you think of any other actions that'd need to reset the `copied` state? I figured these were the main ones but I'm not too sure of the underlying details.